README.md

```diff
@@ -30,8 +30,10 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221180261-77dcacc9-d93e-40ae-9d00-647257390c76.png)

**[Optional]**

- [ ] Kindly delete the two lines after the first heading which are unnecessary in your MIT file. Kindly remove them to make your license file professional :+1:
- [ ] Kindly put a website or web application instead of APP TYPE to make your project licensed properly :+1: because now you are not referencing your license file to your current repo :fearful:

Reply:
send me a message on Whatsapp let solve the problem amicably @habasm

Reply:
@jodavid444 I am available at slack. to day my WhatsApp is crashed.

Reply:
ok am available there.